environment.yml

```diff
@@ -1,14 +1,16 @@
 channels:
   - conda-forge
+  - pytorch
```

Comment:
I think we will need to wait for the conda-forge pytorch to be updated to 2.0?

Reply:
Pytorch's web suggest this for installing pytorch 2:
```bash
conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia
```
Do you think adding the additional channels will be a problem?

Reply:
As I understand it to release on conda-forge all dependencies need to be available from the conda-forge channel: https://conda-forge.org/docs/maintainer/adding_pkgs.html#avoid-external-dependencies

Reply:
I see, then I guess next release will have to wait for that... We can work on making everything compatible for that moment in the meanwhile.